src/Transition/CreateNewIDFUsingRulesV8_7_0.f90

```diff
@@ -427,7 +425,7 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
                    OutArgs(35) = 'MutuallyExclusive'
                  END IF
 
-              CASE('AIRFLOWNETWORK:DISTRIBUTION:COMPONENT:DUCT')
+             CASE('AIRFLOWNETWORK:DISTRIBUTION:COMPONENT:DUCT')
```

Comment:
Not sure how the line on CurArgs = CurArgs + 4 was deleted but it looks like it should be there.